docs/user-guide/wingen.rst

```diff
@@ -1,392 +1,440 @@
-necessary files in a Linux environment. However, generating the necessary binaries can be greatly simplified by using
-the Cobbler post trigger on the sync command. Below is an example of such a trigger, which prepares the necessary files
-for legacy BIOS mode boot. Boot to UEFI Mode with iPXE is simpler and can be implemented by replacing the first 2 steps
-and several others with creating an iPXE boot menu.
+Supported installation options:
 
-Trigger ``sync_post_wingen.py``:
+* UEFI iPXE install (via ipxe-shimx64.efi, ipxe.efi and wimboot tftp/http)
+* BIOS iPXE install (via ipxe undionly.kpxe and wimboot tftp/http)
+* BIOS PXE install (via syslinux pxelinux.0, linux.c32 and wimboot tftp/http)
+* BIOS PXE install (via grub2 grub.0 and wimboot tftp/http)
+* BIOS PXE install (via windows pxeboot.n12)
 
-* some of the files are created from standard ones (``pxeboot.n12``, ``bootmgr.exe``) by directly replacing one string
-  with another directly in the binary
-* in the process of changing the ``bootmgr.exe`` file, the checksum of the PE file will change and it needs to be
-  recalculated. The trigger does this with ``python-pefile``
-* ``python3-hivex`` is used to modify Windows boot configuration data (BCD). For pxelinux distro boot_loader in BCD,
-  paths to ``winpe.wim`` and ``boot.sdi`` are generated as ``/images/<distro_name>``, and for iPXE with
-  wimboot - ``\Boot``.
-* uses ``wimlib-tools`` to replace ``startnet.cmd startup`` script in WIM image
+Installation Quickstart guide
+#############################
 
-Windows answer files (``autounattended.xml``) are generated using Cobbler templates, with all of its conditional code
-generation capabilities, depending on the Windows version, architecture (32 or 64 bit), installation profile, etc.
-
-startup scripts for WIM images (startnet.cmd) and a script that is launched after OS installation (``post_install.cmd``)
-are also generated from templates
+* ``dnf install python3-pefile python3-hivex wimlib-utils``
+* enable Windows support in settings ``/etc/cobbler/settings.d/windows.settings``:
 
-Post-installation actions such as installing additional software, etc., are performed using the Automatic Installation
-Template (``win.ks``).
+.. code::
 
-A logically automatic network installation of Windows 7 and newer can be represented as follows:
+    windows_enabled: true
 
-PXE + Legacy BIOS Boot
+* Share ``/var/www/cobbler`` via Samba:
 
 .. code-block:: text
 
-    Original files: pxeboot.n12 → bootmgr.exe → BCD → winpe.wim → startnet.cmd → autounattended.xml
-    Cobbler profile 1: pxeboot.001 → boot001.exe → 001 → wi001.wim → startnet.cmd → autounatten001.xml → post_install.cmd profile_name
-    ...
+    vi /etc/samba/smb.conf
+            [DISTRO]
+            path = /var/www/cobbler
+            guest ok = yes
+            browseable = yes
+            public = yes
+            writeable = no
+            printable = no
```

Comment:
Can we automate this step maybe somehow? I have never used samba before.

Reply:
Yes, we can automate the creation of smb.conf in the same way as we did for dhcpd.conf. I'll try to do this in the next PR.